zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -656,6 +686,17 @@ def do_convert_data(slack_zip_file: str, realm_subdomain: str, output_dir: str,
     logging.info('######### DATA CONVERSION FINISHED #########\n')
     logging.info("Zulip data dump created at %s" % (output_dir))
 
+def save_avatars(avatar_list: List[ZerverFieldsT], output_dir: str, realm_id: int) -> List[ZerverFieldsT]:
+    for avatar in avatar_list:
+        path = ('%s.png' % (user_avatar_path_from_ids(avatar['user_profile_id'], realm_id)))
+        original_image_path = output_dir +  path
+        urllib.request.urlretrieve(avatar['path'], original_image_path)
```

Comment:
Is `urllib.request.urlretrieve` faster than using `requests`?

Reply:
I didn't find anything to compare how fast both the methods are, however I found out that
`urllib.request.urlretrieve` is a legacy interface in Python 3, and it may be deprecated. Shifting to `requests` would be better. Source :https://docs.python.org/3/library/urllib.request.html#legacy-interface